boards/arm/thingy91_nrf9160/Kconfig.board

```diff
@@ -40,6 +40,15 @@ config THINGY91_STATIC_PARTITIONS_LWM2M_CARRIER
 	help
 	  Use a partition layout including a storage partition needed for the lwm2m carrier library.
 
+config THINGY91_PARTITION_MANAGER
+	bool "Default partition manager logic [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	   Don't enforce static partition layout, but use the default Partition Manager logic to
```

Comment:
```suggestion
	   Don't enforce static partition layout, but use the dynamic Partition Manager logic to
```

Reply:
When looking at this again now, I don't think there's any point in mentioning Partition Manager here at all. So have updated the help text to: _«Disable pre-defined static partition layout. This allows the application to use dynamic layout or define a custom static partition layout for the application...»_